.travis.yml

```diff
@@ -8,12 +8,13 @@ services:
  - docker
 
 before_script:
- - git diff --name-only --exit-code
  - docker build -t studio-frontend --no-cache .
 
 script:
   - npm run lint
   - npm run test
+  - make i18n.pre_validate
```

Comment:
Does this mean translations are run every time something goes to Travis?

Reply:
Yeah, it fails the build if you haven't run `make i18n.docker` before committing, similar to how `make package-lock.validate` will fail if you haven't committed package-lock updates.